### ISSUE TITLE
Cannot mod individual posts

### ISSUE THREAD

**Issue by therealjr** [DangerOnTheRanger/maniwani#168]
What are the features of an individual mod supposed to be? currently when I'm logged in I cannot actually do anything like ban a user or delete a post.

**Comment by DangerOnTheRanger** [comment]
Moderators should be able to delete posts, so it's fairly that you can't. Do you see a moderation menu next to each individual post? It could be a bug (probably is), but attach a screenshot of what the UI looks like for you, if possible.

**Comment by therealjr** [comment]
If the feature is missing I'm sure I can implement it on its own. I just wanted to upgrade my code to your latest release before I continued working.

Here's what I currently see while logged into a slip with admin status: https://ibb.co/HhfLgk5

**Comment by DangerOnTheRanger** [comment]
Yeah, that seems like a bug to me, since the code to delete posts is definitely there, but apparently broken (see [threads.py](https://github.com/DangerOnTheRanger/maniwani/blob/f8f850c6a1efb7d43589ee227caa8f2f70e9fc1f/blueprints/threads.py#L181) and [Post.jsx](https://github.com/DangerOnTheRanger/maniwani/blob/master/frontend/src/components/Post.jsx#L155).
